exec/java-exec/src/main/java/org/apache/drill/exec/store/sys/ExtendedOptionIterator.java

```diff
@@ -134,19 +168,36 @@ public ExtendedOptionValueWrapper next() {
    */
   public static class ExtendedOptionValueWrapper {
 
+    private static final String SHORT_DESCRIPTION_SUFFIX = ".short";
+    private static final int SHORT_DESCRIP_MAX_SIZE = 110;
     public final String name;
     public final String kind;
     public final OptionValue.AccessibleScopes accessibleScopes;
     public final String val;
+    public final Status status;
     public final OptionScope optionScope;
+    public final String description;
 
-
-    public ExtendedOptionValueWrapper(final String name, final String kind, final OptionValue.AccessibleScopes type, final String value, final OptionScope scope) {
+    public ExtendedOptionValueWrapper(final String name, final String kind, final OptionValue.AccessibleScopes type, final String value, final Status status, final OptionScope scope) {
       this.name = name;
       this.kind = kind;
       this.accessibleScopes = type;
       this.val = value;
+      this.status = status;
       this.optionScope = scope;
+      this.description = getShortDescription(name);
+    }
+
+    private String getShortDescription(String name) {
```

Comment:
Please add comment why we might need short description.

Reply:
👍

Reply:
Has comment been added?

Reply:
Looks like I missed this. Done in the latest commit.